tests/storage_tests.cpp

```diff
@@ -280,3 +280,41 @@ TEST_CASE("issue880") {
     Inversion inversion;
     storage.has_dependent_rows(inversion);
 }
+
+namespace {
+    class Record final {
```

Comment:
nit: one can place `Record` right inside the test in case other test will also have a different `Record` class